core/sql/regress/compGeneral/EXPECTED042

```diff
@@ -1977,25 +1977,31 @@ NUM_HITS    NUM_PARAMS
 ----------  ----------
 
          0           1
+         0           1
+         0           1
+         0           1
+         0           2
          0           2
+         0           2
+         0           2
+         0           3
          0           3
+         0           4
          0           5
          1           1
          1           1
          1           2
          1           2
          1           3
+         1           3
```

Comment:
This looks more serious. We have more entries in the query cache, but are getting fewer hits. I wonder what happened.